Sources/TestsCommon/APMUtils.swift

```diff
@@ -0,0 +1,125 @@
+import Foundation
+import MongoSwift
+
+public class TestCommandEventHandler: CommandEventHandler {
```

Comment:
A useful event handler for testing purposes.

Reply:
I think some brief documentation about the types in this file would be useful to future developers